frontend/src/app/modules/social-page/filter-section/filter-section.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="filters-section-container">
+    <div class="btns-wrp">
+        <app-dropdown [isSingleChoice]="true" [btnLabel]="this.compatibilityBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.compatibilities" (selectedFilters)="update('compatibility', $event)"></app-dropdown>
+        <app-dropdown [isSingleChoice]="true" [btnLabel]="this.langBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.languages" (selectedFilters)="update('lang', $event)"></app-dropdown>
+        <app-dropdown [btnLabel]="this.levelBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.langLevels" (selectedFilters)="update('level', $event)"></app-dropdown>
+        <app-dropdown [btnLabel]="this.topicsBtnLabel" [resetEvent]="this.resetFiltersEvent" [data]="this.topics" (selectedFilters)="update('topic', $event)"></app-dropdown>
+    </div>
+    <div class="selected-filters-wrp" *ngIf="selectedLanguagesFilters.size
+         || selectedCompatibilityFilters.size
+         || selectedLevelFilters.size
+         || selectedTopicsFilters.size">
+        <div class="selected-filters-list">
+            <div style="margin: 0;" *ngFor="let selectedCompatibility of this.selectedCompatibilityFilters">
+                <div *ngIf="selectedCompatibility" class="selected-filter-item">
+                    {{selectedCompatibility}}
+                    <mat-icon (click)="remove('compatibility', selectedCompatibility)">close</mat-icon>
+                </div>
+            </div>
+            <div style="margin: 0;" *ngFor="let selectedLang of this.selectedLanguagesFilters">
+                <div *ngIf="selectedLang" class="selected-filter-item">
+                    {{selectedLang}}
+                    <mat-icon (click)="remove('lang', selectedLang)">close</mat-icon>
+                </div>
+            </div>
+            <div style="margin: 0;" *ngFor="let selectedLevel of this.selectedLevelFilters">
+                <div *ngIf="selectedLevel" class="selected-filter-item">
+                    {{selectedLevel}}
+                    <mat-icon (click)="remove('level', selectedLevel)">close</mat-icon>
+                </div>
+            </div>
+            <div *ngFor="let selectedTopic of this.selectedTopicsFilters">
+                <div *ngIf="selectedTopic" class="selected-filter-item">
+                    {{selectedTopic}}
+                    <mat-icon (click)="remove('topic', selectedTopic)">close</mat-icon>
+                </div>
+            </div>
+        </div>
+        <button class="close-filters-btn" (click)="resetFilters()">
+            Close all
+        </button>
```

Comment:
Make it shown only if there are filter items

Reply:
Why not?

Reply:
It's already done